src/fragments/lib/geo/js/geofences.mdx

```diff
@@ -6,6 +6,49 @@
 
 ## Provisioning geofence resources
 
-## Maplibre
+## Manage Geofences in Your Application
+
+First, make sure you've provisioned a geofence collection resource and configured your app using the instructions in either [Amplify CLI - Geo - Geofence Collection](/cli/geo/geofences) or [Use existing Amazon Location Service resources](/lib/geo/existing-resources) and you have already setup [displaying a map](/lib/geo/maps) in your application.
+
+To add a geofence management component to your map, you can use the [amplify-geofence-control](https://github.com/aws-amplify/maplibre-gl-js-amplify/) found in the `https://github.com/aws-amplify/maplibre-gl-js-amplify/` library.
+
+Install the necessary dependencies with the following command:
+
+```bash
+npm install -S maplibre-gl@1 maplibre-gl-js-amplify
+```
+
+> **Note:** Make sure that `maplibre-gl-js-amplify` version `1.2.0` or above is installed.
+
+First, create a map onto which you want to add the geofence management component. See the guide on [creating and displaying maps](https://docs.amplify.aws/lib/geo/maps/q/platform/js/).
+
+Then, use `AmplifyGeofenceControl` to get a new instance of the control and add it to your MapLibre map.
+
+> **Note:** Ensure that your package bundler (webpack, rollup, etc) is configured to handle css files. Check out the webpack documentation [here](https://webpack.js.org/loaders/css-loader/).
+
+```javascript
+import { createMap, AmplifyGeofenceControl } from "maplibre-gl-js-amplify";
+import maplibregl from "maplibre-gl";
+import "maplibre-gl/dist/maplibre-gl.css";
+
+async function initializeMap() {
+    const el = document.createElement("div");
+    el.setAttribute("id", "map");
+    document.body.appendChild(el);
+
+    const map = await createMap({
+        container: "map",
+        center: [-123.1187, 49.2819], // [Longitude, Latitude]
+        zoom: 11,
+    })
+
+    const control = new AmplifyGeofenceControl({})
+    map.addControl(control);
+}
+
+initializeMap();
+```
+
+![A Geofences Management Application](/images/geofence-management-app.png)
```

Comment:
We should clean up this image to show only 2 or 3 geofences with proper names. Also would be good to show both circular and polygon geofences and editing/deleting workflow.

Reply:
The only configuration option for now would be to change the `geofenceCollectionId`, will document that in the API.md.

Will update the example to have one screenshot showing the edit workflow with a circle and one screenshot with the delete workflow for a polygon